application/account-management/WebApp/tailwind.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{ts,tsx}", "./index.html"],
+  content: ["./src/**/*.{ts,tsx}"],
```

Comment:
```suggestion
  content: ["./src/**/*.{ts,tsx}", "./public/index.html"],
```

This way any Tailwind utility classes used in the index html will be added to the css bundle.